adoc/chapters/programming_interface.adoc

```diff
@@ -6511,15 +6511,15 @@ access an element produces undefined behaviour.
 
 ==== Buffer accessor for commands
 
-The [code]#accessor# class provides access to data in a [code]#buffer# from
-within a <<sycl-kernel-function>> or from within a <<host-task>>.
-When used in a <<sycl-kernel-function>>, it accesses the contents of the buffer
-via the device's <<global-memory>>.
-These two forms of the accessor are distinguished by the [code]#AccessTarget#
-template parameter as shown in <<table.accessors.command.buffer.capabilities>>.
-Both forms support the following values for the [code]#AccessMode# template
-parameter: [code]#access_mode::read#, [code]#access_mode::write# and
-[code]#access_mode::read_write#.
+The [code]#accessor# class provides access to data in a [code]#buffer# in three
+different ways.
+It can be used to access the buffer's data from within a
+<<sycl-kernel-function>> via the device's <<global-memory>>.
+It can also be used to access the buffer's data on host from within a host task.
+Finally, it can be used to get a native backend handle to the buffer from within
+a host task, to be used on device.
```

Comment:
The phrase "to be used on device" seems confusing to me.  The native backend handle is used on the host, not on the device.  The handle may refer to data that lives on the device, but the handle itself is used on the host.  I think we could just remove this phrase (and also in the table describing `target::device` which has the same phrase).  Later on, you have this text which clarifies exactly how the handle can be used:

> the accessor may only be used to get a native backend handle for the buffer as described in <<subsec:interfaces.hosttasks.interophandle>>

Reply:
Thanks have rectified